docs/conformance.md

```diff
@@ -1,14 +1,44 @@
 # Conformance testing
 
+The Conformance tests are subset of our e2e test suite that defines set of features
+defining Kubevirt core functionality. The goal of the suite is to make sure that selected features work across different Vendors, CNIs, CSIs, CRIs implementations.
+
+## How are Conformance tests selected
+
+By default tests with a `[Conformance]` tag (and from release 0.59 also decorator) will be executed and tests with a
+`[Disruptive]` tag will be skipped. These tests will not destroy or modify
+anything outside of the explicitly created test namespaces by the binary.
+
+The tests are always part of Kubevirt e2e tests that are usually approved by individual Sigs. For the Conformance tests we require higher requirments than than 
+regular tests, most significant requirments are:
+- Test is not Disruptive
+- Test is not flaky and is reliable per data available from CI team
+- For more concreate details see below
+
+
+## Conformance tests requirments:
+- The feature being tested is GA (no feature gated)
+- The feature is not deprecated
+- The feature is turned on by default (no feature toogle)
```

Comment:
An exception to this could be upgrade test. WDYT?

Reply:
I am not sure upgrade tests are eligible as they do modify the installation. I believe that has to be leave for vendors, users to tests for themself...

Reply:
```suggestion
- The feature is turned on by default (no feature toggle)
```